src/app/components/work-item-quick-add/work-item-quick-add.component.less

```diff
@@ -94,10 +96,11 @@
     margin-left: 0;
     background: @color-pf-white;
     .Border(2px, solid, @color-pf-black-400);
-    textarea[class="f8-quickadd-desc"] {
+    textarea.f8-quickadd-desc {
```

Comment:
why change of syntax?

Reply:
`textarea[class='f8-quickadd-desc']` isn't the correct way of selecting a textarea with class f8-quickadd-desc. If you use `textarea[class='f8-quickadd-desc']` none of the styles are reflected.